pool/routes/routes.ml

```diff
@@ -555,10 +555,7 @@ module Admin = struct
       in
       let models =
         [ get "" ~middlewares:[ Access.index ] index
-        ; post
-            "sort-fields"
-            ~middlewares:[ Access.sort_ungrouped ]
-            sort_ungrouped_fields
+        ; post "sort-fields" ~middlewares:[ Access.sort ] sort_ungrouped_fields
```

Comment:
This middleware does not work (also sort groups). They both throw `Not_found` exception. They both look for an id in the route.

Reply:
refactored